pyiso/isone.py

```diff
@@ -188,3 +204,66 @@ def parse_json_load_data(self, data):
                 return data['FiveMinSystemLoads']['FiveMinSystemLoad']
         except (KeyError, TypeError):
             raise ValueError('Could not parse ISONE load data %s' % data)
+
+    def parse_json_lmp_data(self, data):
+        """
+        Pull approriate keys from json data set.
+        Raise ValueError if parser fails.
+        """
+        try:
+            if self.options.get('latest'):
+                return data['FiveMinLmp']
+            else:
+                return data['FiveMinLmps']['FiveMinLmp']
+        except (KeyError, TypeError):
+            raise ValueError('Could not parse ISONE lmp data %s' % data)
+
+    def get_lmp(self, zone_name, latest=True, start_at=False, end_at=False, **kwargs):
+        # set args
```

Comment:
Refactored to reuse ISONE helper methods. Should do exactly the same thing, just different return type